app/views/alchemy/admin/pictures/_picture_to_assign.html.erb

```diff
@@ -18,7 +12,7 @@
     action_url,
     remote: true,
     onclick: '$(self).attr("href", "#").off("click"); return false',
-    method: action_method,
+    method: @action_method,
```

Comment:
Yeah, logic in the views, vs. instance var, don't know what's the right way to do here. 
It was like this, see the diff above ;)

But, maybe your'e right